_posts/en/newsletters/2021-03-31-newsletter.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Bitcoin Optech Newsletter #142'
+permalink: /en/newsletters/2021/03/31/
+name: 2021-03-31-newsletter
+slug: 2021-03-31-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a paper and a short discussion about
+probabilistic path selection for LN and includes our regular sections with
+summaries of popular questions and answers from the Bitcoin
+StackExchange, release and release candidates, and notable changes to
+Bitcoin infrastructure software.
+
+## News
+
+- **Paper on probabilistic path selection:** René Pickhardt
+  [posted][pickhardt post] to the Lightning-Dev mailing list a
+  [paper][pickhardt et al] he co-authored with Sergei Tikhomirov, Alex
+  Biryukov, and Mariusz Nowostawski.  The paper models a network of
+  channels having a uniform distribution of balances within their
+  respective channel capacity.  E.g., for a channel with the capacity of
+  100 million satoshis between Alice and Bob, the paper assumes all of
+  the following states are equally as likely for that channel, and that
+  the same holds true for every other channel on the network:
+
+    | Alice | Bob |
+    | 0 sat | 100,000,000 sat |
+    | 1 sat | 99,999,999 sat |
+    | ... | ...|
+    | 100,000,000 sat | 0 sat |
+
+   Making this assumption allows the authors to draw conclusions about
+   the probability that a payment will succeed based on its amount and how
+   many hops (channels) it needs to traverse.  This allows them to
+   prove the benefit of several known heuristics---such as keeping paths
+   short and using [multipath payments][topic multipath payments] to break
+   larger payments into smaller payments (under certain other
+   assumptions).  They also use the model to evaluate new proposals,
+   such as allowing [Just-In-Time (JIT) rebalancing][topic jit routing]
+   via [bolts #780][].
+
+   The paper uses its conclusions to provide a routing algorithm that it
+   claims can reduce payment retry attempts by 20% compared to their
+   simplification of existing routing algorithms.  The new algorithm
+   prefers routes with a higher computed probability of success, whereas
+   existing algorithms use a heuristic approach.  Combined with JIT
+   rebalancing, they estimate a 48% improvement.  Given that each retry
+   usually requires several seconds, and could take much longer in some
+   cases, this could provide a significant benefit to the network.
```

Comment:
I'm not sure what "benefit to the network" means here. Would "improved user experience" be better or is it broader than that?